src/Microsoft.PowerShell.ConsoleGuiTools/ConsoleGui.cs

```diff
@@ -27,10 +27,11 @@ public HashSet<int> Start(ApplicationData applicationData)
             _applicationData = applicationData;
             _gridViewDetails = new GridViewDetails
             {
-                // Have a 8 character addition of a checkbox ("     [ ]") that we have to factor in.
-                ListViewOffset = 8
+                // If we have an OutputMode, then we want to make them selectable. If we make them selectable,
+                // they have a 8 character addition of a checkbox ("     [ ]") that we have to factor in.
+                ListViewOffset = _applicationData.OutputMode != OutputModeOption.None ? 8 : 4
             };
-
+            
```

Comment:
Please revert the change.

You need to turn on a feature to remove trailing spaces in your editor.